packages/screens/DAppStore/apps/toripunks/content/RussianRoulette.tsx

```diff
@@ -177,6 +181,15 @@ export const Russian = () => {
       return setResult(!result);
     }
     if (!bet) return;
+    if (balance.length > 0) {
+      if (
+        balance[0].denom === "utori" &&
```

Comment:
if user does not have utori then we by pass this check ? 
According to this check, we check only if denom = utori, is that expected behaviour ?

Reply:
yes, the toripunks app only cares about TORI